services/login.webmaker.org/app/http/controllers/site.js

```diff
@@ -38,14 +38,16 @@ exports.js = function (filename) {
  * GET health check for app
  */
 var version = require("../../../package").version;
+var commit = require("../../../node_modules/git-rev-sync/").long;
```

Comment:
Same thing here, no pathing to get to `git-rev-sync`

Reply:
undo changes in this file...we shouldn't update services in here.